alerting/dashboardunifie/README.md

```diff
@@ -0,0 +1,85 @@
+# ALERTING Dashboard Unifie SignalFx integrations
```

Comment:
I would prefer `UnifiedDashboard` is ok for you?

Reply:
Changed